docker/python2/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.9
+
+ARG VERSION
+
+RUN apk add --no-cache python2 py2-pip && \
```

Comment:
Could we just base this on the python image instead of bare alpine? Then we wouldn't have to install python:

```
From python:2.7-alpine
```